src/js/models/interfaces/index.ts

```diff
@@ -56,6 +56,8 @@ export type Post = {
   watch: number;
   peak?: Peak;
   user?: User;
+  latitude: number;
```

Comment:
Typy opcjonalne raczej nie każdy szczyt ma współrzędne

Reply:
W bazie każdy ma, ale mogę zmienić na opcjonalne

Reply:
Właściwie to każdy post ma współrzędne, tylko jak nie były zdefiniowane to są ustawione na 0.0, 0.0

Reply:
Po stronie backendu tak się robi